collector/filesystem_linux.go

```diff
@@ -87,9 +88,17 @@ func mountPointDetails() ([]filesystemLabels, error) {
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
 		parts := strings.Fields(scanner.Text())
+
+		// Ensure we handle the translation of \040 and \011
+		// as per fstab(5)
+		mp, err := strconv.Unquote(parts[1])
+		if err != nil {
+			continue
```

Comment:
This is ignoring the error, we don't want to do that. I would have it return the `err` if there is one like is done on line 82.